docs/developer/extending/apps/key-concepts.mdx

```diff
@@ -140,6 +140,9 @@ The structure of `targetUrl` should be as below:
 awssqs://<access_key_id>:<secret_access_key>@sqs.<region>.amazonaws.com/<account_id>/<queue_name>
 ```
 
+:::note
+FIFO queues (ending in `.fifo`) must be configured to have `ContentBasedDeduplication` enabled.
```

Comment:
Should we document what the grouping key would be set to?

Reply:
Done :)